code/modules/mob/living/carbon/human/species.dm

```diff
@@ -41,7 +41,7 @@
 	var/say_mod = "says"	// affects the speech message
 	var/list/default_features = list() // Default mutant bodyparts for this species. Don't forget to set one for every mutant bodypart you allow this species to have.
 	var/list/mutant_bodyparts = list() 	// Parts of the body that are diferent enough from the standard human model that they cause clipping with some equipment
-
+	var/list/mutant_organs = list(/obj/item/organ/internal/tongue)		//Internal organs that are unique to this race.
```

Comment:
Why `zone = "head"` when you can actually put them in mouth? It's supported by some of existing organs and organ manipulation for mouth is already a thing.